TabPanel: Add Overview (#9078)
DevExpress/devextreme-documentation: concepts/05 UI Components/TabPanel/02 Overview/10 Key Features.md

- **Data Binding**    
TabPanel supports different [data source](/Documentation/ApiReference/Data_Layer/DataSource/) types and can update displayed items when data is modified.

    #include btn-open-demo with {
        href: "https://js.devexpress.com/Demos/WidgetsGallery/Demo/TabPanel/DragAndDrop/"
    }

- **Layout Configuration**    

> The component can stack tabs in both horizontal and vertical directions. TabPanel also allows you to [position](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#tabsPosition) tabs along any edge of the component (top or bottom, left or right).

Reviewer: ```suggestion
The component can arrange tabs horizontally or vertically. TabPanel also allows you to [position](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#tabsPosition) tabs along any edge of the component: top, bottom, left, or right.
```

- **Accessibility and Keyboard Navigation**    
TabPanel [conforms](/Documentation/Guide/UI_Components/TabPanel/Accessibility/) to WCAG 2.x, European Accessibility Act (EAA), and Americans with Disabilities Act (ADA) standards. The component supports [keyboard navigation](/Documentation/Guide/UI_Components/TabPanel/Accessibility/#Keyboard_Navigation) and custom key handlers. TabPanel also ships with right-to-left ([RTL](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#rtlEnabled)) representation support.

- **Customization**    

> You can use templates to define custom markup for TabPanel tabs ([specific tabs](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/items/#tabTemplate) and [all tabs](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#itemTitleTemplate)) as well as panels ([specific panels](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/items/#template) and [all panels](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#itemTemplate)). The component also allows you to toggle [swipe](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#swipeEnabled) gestures and transition [animations](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#animationEnabled) to customize how users switch between tabs.

Reviewer: ```suggestion
You can use templates to define custom markup for individual TabPanel tabs ([specific tabs](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/items/#tabTemplate) or all tabs [all tabs](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#itemTitleTemplate)), and for individual panels ([specific panels](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/items/#template) or all panels [all panels](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#itemTemplate)). You can also enable or disable [swipe](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#swipeEnabled) gestures and transition [animations](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#animationEnabled) to customize how users switch between tabs.
```
